examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
I'm ok witha  complex option, it's mostly that the logstash config parser stops offering validation at the "Is it a hash? OK, cool" boundary.

You can specify a custom `:validate => lambda { ... }` to provide your own custom validation, but that hasn't been used in much of any setting in quite some time. Still, _some_ kind of validation with helpful error messages is important to help guide users to correct configurations.